astrolabe/utils.py

```diff
@@ -261,14 +264,14 @@ def read_stats(self):
             LOGGER.info("Reading sentinel file {!r}".format(self.sentinel))
             with open(self.sentinel, 'r') as fp:
                 stats = json.load(fp)
+                LOGGER.info("Sentinel contains: %s" % json.dumps(stats))
```

Comment:
I found this was helpful for debugging, since this file is never reported elsewhere and also does not get saved as a build artifact like `events.json`. That said, saving `events.json` isn't very helpful for the validation tests since only the final copy of it will be persisted.